token/src/state.rs

```diff
@@ -6,9 +6,13 @@ use crate::{
     option::COption,
 };
 use solana_sdk::{
-    account_info::next_account_info, account_info::AccountInfo, entrypoint::ProgramResult, info,
+    account_info::next_account_info, account_info::AccountInfo, entrypoint::ProgramResult,
     program_error::ProgramError, pubkey::Pubkey,
 };
+#[cfg(feature = "program")]
```

Comment:
Side change to help logging in non-bpf environments?

Reply:
Yes I'll make the solana-sdk changes so that cb5f1e245fe285cbd688f1cd79c2f51f4bcae27a won't need to be merged